src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationService.java

```diff
@@ -30,10 +43,24 @@ public void processServiceBusMessage(ErrorMsg message) {
             message.errorDescription,
             message.id
         );
+        ErrorNotification entity = new ErrorNotification(
+            entityManager.getReference(ProcessEvent.class, message.eventId),
+            message.errorCode.name(),
+            message.documentControlNumber,
+            message.id
+        );
+
+        try {
+            ErrorNotificationResponse response = client.notify(request);
+
+            entity.setNotificationId(response.getNotificationId());
 
-        ErrorNotificationResponse response = client.notify(request);
+            log.info("Error notification published. ID: {}", response.getNotificationId());
+        } catch (Exception exception) {
+            log.error("Failed to publish error notification. {}", exception.getMessage(), exception);
```

Comment:
Adding exception message to the error message is unnecessary. It will be there in the log, anyway.

Reply:
oh will it? i vaguely remember missing those messages. but perhaps i'm confusing with something else

Reply:
There used to be a problem, but it's solved now.